browser/ui/views/brave_actions/brave_actions_container.cc

```diff
@@ -353,6 +353,15 @@ bool BraveActionsContainer::CanStartDragForView(View* sender,
 }
 // end ToolbarActionView::Delegate members
 
+// BraveActionsContainerDelegate members
+void BraveActionsContainer::OnPopupClosed(
```

Comment:
seems like there must be a better way to handle this cc @petemill

Reply:
This change was done basing on comment from here:
https://github.com/brave/brave-core/blob/4384e9425c5b56953f68469f16d224e1aba98859/browser/ui/views/brave_actions/brave_actions_container.cc#L209